features/FEATURE_BLE/ble/BLETypes.h

```diff
@@ -411,32 +411,6 @@ struct byte_array_t {
     uint8_t _value[array_size];
 };
 
-/**
- * Construct a fixed size ArrayView from a byte_array_t.
- *
- * @param src byte_array_t to create a view from.
- *
- * @return An ArrayView to @p src.
- */
-template<size_t Size>
-ArrayView<uint8_t, Size> make_ArrayView(byte_array_t<Size>& src)
```

Comment:
I need to add it back as make_Span(byte_array_t<Size>&), because nRF drivers need it. Same with make_const_Span().